developer-concepts/readme.adoc

```diff
@@ -837,9 +841,9 @@ Deleting a job deletes all the pods as well. Delete the job as:
 
 === Pre-requisites
 
-For Kubernetes cluster versions < 1.8, Cron Job is created with API version `batch/v2alpha1`. This version needs to be explicitly enabled in the Kubernetes cluster as explained in link:cluster-install#turn-on-an-api-version-for-your cluster[Turn on an API version for your cluster].
+For Kubernetes cluster versions < 1.8, Cron Job is created with API version `batch/v2alpha1`. This version needs to be explicitly enabled in the Kubernetes cluster as explained in link:../cluster-install#turn-on-an-api-version-for-your-cluster[Turn on an API version for your cluster].
 
-`batch/v2alpha1` API version for Cron Job is deprecated for the cluster version >= 1.8. You should switch to `batch/v1beta1`. This version is enabled by default in the API server.
+`batch/v2alpha1` API version for Cron Job is deprecated for the cluster version >= 1.8. You should switch to `batch/v1beta1`. Keep in mind, once you switch API versions, you need to perform rolling-update of the cluster which generally takes 30 - 45 mins to complete. 
```

Comment:
Highlight the last sentence as a NOTE: ?

Also, the time depends upon the number of nodes in the cluster. Its typically 5-7 mins/node.

Reply:
fixed in newer commit